relentless/simulate/lammps.py

```diff
@@ -512,107 +582,135 @@ def to_commands(self, sim):
+             (self.barostat is None or isinstance(self.barostat, simulate.BerendsenBarostat))):
+            cmds = ['fix {idx} {group_idx} nvt temp {Tstart} {Tstop} {Tdamp}'.format(idx=self._fix,
+                                                                                     group_idx='all',
+                                                                                     Tstart=self.thermostat.T,
+                                                                                     Tstop=self.thermostat.T,
+                                                                                     Tdamp=self.thermostat.tau)]
+            if isinstance(self.barostat, simulate.BerendsenBarostat):
+                fix_berendsen_pres = True
+        elif ((self.thermostat is None or isinstance(self.thermostat, simulate.BerendsenThermostat)) and
+              isinstance(self.barostat, simulate.MTKBarostat)):
+            cmds = ['fix {idx} {group_idx} nph iso {Pstart} {Pstop} {Pdamp}'.format(idx=self._fix,
+                                                                                    group_idx='all',
+                                                                                    Pstart=self.barostat.P,
+                                                                                    Pstop=self.barostat.P,
+                                                                                    Pdamp=self.barostat.tau)]
+            if isinstance(self.thermostat, simulate.BerendsenThermostat):
+                fix_berendsen_temp = True
+        elif isinstance(self.thermostat, simulate.NoseHooverThermostat) and isinstance(self.barostat, simulate.MTKBarostat):
+            cmds = ['fix {idx} {group_idx} npt temp {Tstart} {Tstop} {Tdamp} iso {Pstart} {Pstop} {Pdamp}'.format(idx=self._fix,
+                                                                                                                  group_idx='all',
+                                                                                                                  Tstart=self.thermostat.T,
+                                                                                                                  Tstop=self.thermostat.T,
+                                                                                                                  Tdamp=self.thermostat.tau,
+                                                                                                                  Pstart=self.barostat.P,
+                                                                                                                  Pstop=self.barostat.P,
+                                                                                                                  Pdamp=self.barostat.tau)]
+        else:
+            raise TypeError('An appropriate combination of thermostat and barostat must be set.')
+
+        if fix_berendsen_temp:
+            _fix_berendsen_t = super().new_fix_id()
```

Comment:
Will this not work as?
```suggestion
            _fix_berendsen_t = self.new_fix_id()
```